.github/workflows/publish.yaml

```
@@ -0,0 +1,40 @@
+name: publish
+
+on:
+  push:
+    branches:
+      - main
```

Use square brackets for list.
```suggestion
    branches: [main]
```